### ISSUE TITLE
Can “buildGraph”functions return pyg format?

### ISSUE THREAD

**Issue by ljy278** [KevinHoarau/BML#6]
Now buildGraph function return networkx format. Can I modify it to return in Pyg format?

**Comment by KevinHoarau** [comment]
Yes you can do it using the [from_networkx](https://pytorch-geometric.readthedocs.io/en/latest/modules/utils.html#torch_geometric.utils.from_networkx) function of the pyg library.
Here an [example](https://github.com/KevinHoarau/BGNN/blob/main/transform_to_pyg.py) in a previous work.

**Comment by ljy278** [comment]
Yes, but it's too slow.

**Comment by KevinHoarau** [comment]
The example is slow because it is single-threaded, however, if you use from_networkx in the BML's transformation then it could be parallelized. But maybe you already tried, and it is still too slow.
